qwik-graphql-tailwind/src/components/repo-about/description/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { component$ } from '@builder.io/qwik';
+
+interface DescriptionProps {
+  text?: string | null;
```

Comment:
the `?` means it could be undefined or null, so not sure why its there when there is a null among the possible value

Reply:
Most of the interfaces were copied from the other projects